Update the documentation for Next on Workers (#20908)
cloudflare/cloudflare-docs: src/content/docs/workers/frameworks/framework-guides/nextjs.mdx

	WranglerConfig
} from "~/components";

> A more detailed documentation is available on the [Open Next website](https://opennext.js.org/cloudflare).
> 

Reviewer: I think that here we should start saying that this guide is about (also for consistency in respect to the other framework guides) also I think it would be beneficial to clarify when to follow this guide and when to instead go to the OpenNext documentation

What about something like:
```suggestion
In this guide, you will create a new [Next.js](https://nextjs.org/) application and deploy to Cloudflare Workers (with the new [<InlineBadge preset="beta" /> Workers Assets](/workers/static-assets/)) using the [`@opennextjs/cloudflare`](https://opennext.js.org/cloudflare) package.

:::note
This is a simple getting started guide for a more detailed documentation on how the to use the Cloudflare Open Next adapter visit the [Open Next website](https://opennext.js.org/cloudflare).
:::
```
?

Add the following to the scripts field of your `package.json` file:

```json
"preview": "opennextjs-cloudflare && wrangler dev",
"deploy": "opennextjs-cloudflare && wrangler deploy",
"cf-typegen": "wrangler types --env-interface CloudflareEnv cloudflare-env.d.ts"
```

- `preview`: Builds your app and serves it locally, allowing you to quickly preview your app running locally in the Workers runtime, via a single command.
- `deploy`: Builds your app, and then deploys it to Cloudflare
- `cf-typegen`: Generates a `cloudflare-env.d.ts` file at the root of your project containing the types for the env.

### 4. Optionally add caching

Caching is actively being worked on. It is fully functional for development and we are working on an optimized implementation suitable for production.

> Check [the documentation](https://opennext.js.org/cloudflare/caching) on the Open Next website.

Reviewer: ```suggestion
For more details check the relevant [official Open Next documentation](https://opennext.js.org/cloudflare/caching).
```

### 5. Develop locally

You can continue to run `next dev` when developing locally.

### 6. Preview locally your application and create an OpenNext config file

> In step 3, we also added the `npm run preview`, which allows you to quickly preview your app running locally in the Workers runtime, rather than in Node.js.

Reviewer: ```suggestion
In step 3, we also added the `npm run preview` script, which allows you to quickly preview your app running locally in the Workers runtime, rather than in Node.js.
```
